.gitignore

```diff
@@ -2,4 +2,6 @@ build/
 logs/
 nbproject/
 vendor/
+.idea/
```

Comment:
You can also put it in `.git/info/exclude` instead - which is a local per-repository setting.